android/KMAPro/kMAPro/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,13 @@
       </intent-filter>
     </receiver>
 
+    <!--    see the events from this app in sentry.keyman.com dashboard -->
+    <meta-data
+      android:name="io.sentry.dsn" android:value="https://2f067e5571a74974b9e210ca76ca9607@sentry.keyman.com/7" />
```

Comment:
Is it possible for us to extract the dsn into a separate file? Not sure it will be readily possible given this is an xml but I don't know...

Reply:
> Is it possible for us to extract the dsn into a separate file?

Are you asking if we can configure the DSN in a separate file? The DSN can alternatively be placed in a `sentry.properties` file (I believe at the app root path)

Configuration [reference](https://docs.sentry.io/clients/java/config/). 

> Not sure it will be readily possible given this is an xml...

Can you clarify what your uncertainty is about "readily possible"? 
The ["Connecting the SDK to sentry" guide](https://docs.sentry.io/platforms/android/) says to just add the DSN to the manifest file, so it seems that's the preferred method.

Or if you're asking about extracting the DSN during runtime to export somewhere, I'd have to investigate that...

Reply:
The ideal: have a /resources/sentry.properties JSON file which is consumed by all projects at compile time. This way we can specify details such as sentry server, project, DSN, etc in one place.

The reality: this may not work out too well. No point in adding extra code because that is more to maintain.

So it's a question only: if we can get an easy win DRYing out our references to the DSN, let's do it. Otherwise, we won't worry. I've asked @jahorton the same question.

Reply:
Which is now documented as issue #2792.